### ISSUE TITLE
what is the pre-train model?

### ISSUE THREAD

**Issue by xiaohangguo** [ultralytics/ultralytics#2139]
### Search before asking

- [X] I have searched the YOLOv8 [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.

### Question

hey guy,I using your Ultralytics Hub to training a model,I want to know what is the config of 'pre-trained'？
![QQ截图20230420165832](https://user-images.githubusercontent.com/72799392/233315530-2ef7c6eb-fd3c-41bd-997f-104958e219d3.png)
Is it possible to fine-tune the weight already trained in Coco2017?

**Comment by glenn-jocher** [comment]
Hi @xiaohangguo, thanks for reaching out with your question! The `pretrained` argument in YOLOv8 specifies whether to load a set of pretrained weights, and which weights to load. By default, if `pretrained=True`, YOLOv8 will load the following pretrained weights:

- A General Object Detection model pretrained on COCO from Ultralytics
- A Face Detection model pretrained on WIDERFACE from Ultralytics 

These weights are available for download on the [YOLOv8 Model Zoo](https://github.com/ultralytics/yolov5/wiki/Model-Zoo). You can also load custom weights with `pretrained=path/to/weights.pt`.

Regarding your question about fine-tuning weights already trained on COCO2017, it is possible to do so. You can load the pretrained COCO weights, freeze some of the layers, and continue training on your custom dataset. However, keep in mind that fine-tuning might require careful selection of the layers to fine-tune, as well as finding an appropriate learning rate and schedule.  I hope that helps! Let me know if you have any further questions.

**Comment by xiaohangguo** [comment]
Thank you,you mean that my training is based on this weight model .
You just mentioned that there are two types of pre training models, one based on the COCO dataset and the other based on the WIDE FACE face detection. I did not select either model on the Ultralytics Hub, so which one will the system choose for me by default? Is it the pre trained model for this Coco or this Wide Face? Or is it said that this pre trained model was trained on these two datasets.

**Comment by xiaohangguo** [comment]
The model I returned is still being trained, so I haven't been able to reason yet. So I want to ask if enabling this pre training model configuration means that my model can recognize labels such as buses and people.

**Comment by glenn-jocher** [comment]
@xiaohangguo, that's correct. When you specify `pretrained=True` in YOLOv8, the training will start with a pre-trained model that is based on both the COCO dataset and WIDERFACE. These pre-trained weights can be downloaded from the YOLOv8 Model Zoo, and they contain useful features that detect a wide array of objects, including people and buses which are part of the COCO dataset. However, keep in mind that the accuracy and detection performance of these pre-trained models may vary depending on the specific conditions of your training data.

**Comment by nadinator** [comment]
What if I loaded a model in previously and simply called a .train on that model with pretrained=True? Does that still use the COCO model or will it use my own?

**Comment by pderrenger** [comment]
@nadinator if you loaded a model previously and then called `.train()` on that model with `pretrained=True`, the training will start with the COCO model, not your own. The `pretrained=True` argument overrides any previous weights loaded and starts the training process with the pre-trained COCO model provided by Ultralytics. If you want to use your own weights for training, you should not specify `pretrained=True`.

**Comment by camachoandres88** [comment]
I already trained a yolo11n model with 10k images and my own classes, I recolected 5k images more, if I don't want to start training from scratch, what should I do to add those 5k new images to the training??  It's not clear for me

**Comment by Y-T-G** [comment]
@camachoandres88 Combine with old data

**Comment by camachoandres88** [comment]
@Y-T-G thanks for the answer. Do you know if, apart from that technique, there are any other alternatives? You know that training 15k images will be more expensive than training only 5k. We have already spent some time and computational resources training 10k images, so I would like to know if it is possible to avoid training from scratch

**Comment by morzahavi** [comment]
> [@nadinator](https://github.com/nadinator) if you loaded a model previously and then called `.train()` on that model with `pretrained=True`, the training will start with the COCO model, not your own. The `pretrained=True` argument overrides any previous weights loaded and starts the training process with the pre-trained COCO model provided by Ultralytics. If you want to use your own weights for training, you should not specify `pretrained=True`.

What if I trained a model from scratch on my dataset (with classes different than COCO), and I want to use this model as a pretrained model on my dataset again (to improve performance). 

Is the following code adequate for this purpose:

```python
model = YOLO('path/to/previuos/model.pt') 
model.train(data='dataset.yaml')
``` 

I have used this code, and got MAP-50-95 which is lower than my previous model. How is possible?

**Comment by glenn-jocher** [comment]
@morzahavi yes, your code is correct for using your previously trained model as a starting point for further training. However, a lower mAP50-95 could be due to several factors, such as overfitting, data imbalance, or suboptimal hyperparameters. Ensure your new dataset is well-prepared, and consider adjusting training parameters like learning rate or using fewer frozen layers to better fine-tune your model.
